### ISSUE TITLE
Shouldn't always add http in front of extension, chrome, and edge URLs

### ISSUE THREAD

**Issue by keithx2** [htrinter/Open-Multiple-URLs#88]
These schemes are legal in the following browsers and an extension can create URLs with them:

In Firefox:

moz-extension://

In Chrome:

chrome://
chrome-extension://

In Edge:

chrome:// (redirects to edge://)
edge://
extension://

**Comment by htrinter** [comment]
Fixed in #96. Merged to develop, should be in the extension store soon.

**Comment by 6324** [comment]
> [已在#96](https://github.com/htrinter/Open-Multiple-URLs/pull/96)中修复。已合并到开发中，应该很快就会出现在扩展商店中。

Sorry, I don't understand why I need to remove the "http". Sometimes, most of the addresses I enter are in the format of 192.168.1.1/xx/xxx. If all https are added, these links cannot be opened.

**Comment by htrinter** [comment]
You're right ... a side effect of the recent change. I will add an exemption for IP addresses.

**Comment by htrinter** [comment]
@6324 I guess the problem is that the address you are trying to reach does not support https, only http?

**Comment by 6324** [comment]
> [@6324](https://github.com/6324) I guess the problem is that the address you are trying to reach does not support https, only http?

yes，I think the previous https/http judgment conditions should be retained, but I am no longer in this industry and have no time to contribute code

**Comment by htrinter** [comment]
I reverted back to the http prefixing, for compatibility. Should be in stores soon.
